pkg/cmd/get.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"github.com/hobbyfarm/hfcli/pkg/scenario"
+	command "github.com/rancher/wrangler-cli"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type Get struct{}
+
+func NewGet() *cobra.Command {
+	get := command.Command(&Get{}, cobra.Command{
+		Use:   "get",
+		Short: "get objects, valid options are scenario",
```

Comment:
may be leverage to ensure correct args are supplied. `Args: cobra.ExactArgs(2),`